test/03-error-handling.js

```diff
@@ -1,24 +0,0 @@
-const tap = require('tap')
```

Comment:
Deleted this as I thought there might be the situation where you're adding / linking to an internal library without a `package.json`. Rare but it might happen and most likely we wouldn't want to return the error and just continue.